consumer-features/example-provider-hello-world/v2/example-provider-hello-world-js/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Pact logs?

Maybe should include some guidelines on what should be included in an ignore in the README

Reply:
yeah that is a good shout, normally I would exclude generated pacts from being included, as well as logs, so general guidance on this isn't a bad shout